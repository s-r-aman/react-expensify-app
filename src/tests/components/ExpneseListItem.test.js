import React from 'react';
import { shallow } from "enzyme";

import expenses from "../fixtures/expenses";
import ExpenseListItem from "../../components/ExpenseListItem";

describe('<ExpenseListItem/>', () => {
  test('should render the expense list item correctly with the given data', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]}/>);
    expect(wrapper).toMatchSnapshot();
  });
});