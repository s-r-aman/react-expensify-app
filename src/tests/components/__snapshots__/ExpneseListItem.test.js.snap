// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExpenseListItem/> should render the expense list item correctly with the given data 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    195
     - 
    0
  </p>
</div>
`;
